### PR TITLE
Youtube clone Homepage

### PR DESCRIPTION
**Pull request by cgcarlosg**
This is our youtube cloned page which covers embedding videos and images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # YouTube-video-player-clone
 This is a repository containing a youtube video player clone
+![name-of-you-image](https://github.com/cgcarlosg/YouTube-video-player-clone/blob/feature-branch/images/projectsnapshot.jpg)
+
+Additional description about the project and its features.
+
+Built With
+HTML and CSS
+https://cgcarlosg.github.io/YouTube-video-player-clone/
```
Comment (javitocor) [inline README.md:10]: here you can use this format: `[live demo link](your link)`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+Authors
+
+👤 Author1: carlos gutierrez
+GitHub: @cgcarlosg
+
+👤 Author2: Galiwango Ananiya
+GitHub: @gango-anan
```
Comment (javitocor) [inline README.md]: Add more social media links

File: index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                <li><ion-icon name="apps-outline" id="apps"></ion-icon></li>
+                </ul>
+                </div>
+                <div class="sign">
+                <button type="submit">Sign in</button>
+                </div>
+            </div>
+        </nav>
+        <div class="space"></div>
+    <section class="main">
```
Comment (javitocor) [inline index.html]: you do not need to add an empty div here, you can add a HR tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            </div>
+            <div class="searchbox">
+                <div class="form">
+                <form>
+                    <input type="text" placeholder="Search" id="input">
+                </form>
+                </div>
+                <div class="buttonsearch">
+                <button  id="iconbutton">
+                    <ion-icon name="search-outline"></ion-icon>
```
Comment (javitocor) [inline index.html]: better use classes than  id, it is a better practice

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+    display: flex;
+    font-size: 35px;
+    align-items: center;
+    cursor: pointer;
+}
+#menulines{
+    margin-left: 20px;
+    float: right;
+}
+#youtubelogo{
```
Comment (javitocor) [inline styles.css]: again, change it into classes

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good job for this your first project, guys! 👍 
Good descriptive commits and clean code! 💯 
The website is almost a copy, and you have shown you can use different methods to positioning elements! 👏 

However, still some changes to do and your project will be approved! 💪 
Do not feel discouraged, we are here to learn and all the changes will make your project look more professional! 🧑‍🎓 

Please, check my comments below 👇 , apply the changes and resubmit your project for a new code review ⏩ 

Good luck, guys! 😃 

- Kindly add the footer, as you can see in the link provided. [here](http://archive.vn/Bss88)
- Please, modify your readme file, it is important to have a professional readme, pay attention to the details, you can follow the guidelines from the pathwright, the live link must be hidden and you should also add your LinkedIn and Twitter or other social media to it:
      - The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- Kindly, use classes instead of Id's, it is a better practice since they are reusable.
- Kindly, add cursor pointer to your links so it looks more like the original.

**Comment by cgcarlosg** [comment]
Changes done

**Comment by cgcarlosg** [comment]
Changes done

**Pull request closed without merging**